pd-client/rpc_worker.go

```diff
@@ -312,23 +312,36 @@ func (w *rpcWorker) getClusterConfigFromRemote(conn *bufio.ReadWriter, clusterCo
 	return rsp.GetGetClusterConfig(), nil
 }
 
-func (w *rpcWorker) callRPC(conn *bufio.ReadWriter, req *pdpb.Request) (*pdpb.Response, error) {
+func (w *rpcWorker) callRPC(conn *bufio.ReadWriter, req *pdpb.Request) (resp *pdpb.Response, err error) {
+	// Record some metrics.
+	start := time.Now()
+	label := convertName(req.GetCmdType().String())
+	defer func() {
+		if err == nil {
```

Comment:
It's OK, I learn it from [here](https://blog.golang.org/defer-panic-and-recover).